apps/sms/views/shared/js/settings.js

```diff
@@ -35,14 +36,16 @@ var Settings = {
     var keyHandlerSet = {
       'dom.mms.operatorSizeLimitation': this.initMmsSizeLimitation.bind(this),
       'operatorResource.sms.maxConcat':
-        this.initSmsMaxConcatenatedMsg.bind(this)
+        this.initSmsMaxConcatenatedMsg.bind(this),
+      'messages.mms.sendReadReport.enabled':
```

Comment:
here didn't use constant, because of two reason. (i) make consistency and look better like others two. (ii) it will passed as key i..e Settings[...] in a suite function , so if i use this.SEND_READ_REPORT_KEY. Throwing error in test.